config

```diff
@@ -21,7 +25,10 @@ export NPM_CONFIG_PREFIX=~/.che_node_modules
 export PATH=$NPM_CONFIG_PREFIX/bin:$PATH
 
 export RH_DIST_SUFFIX=fabric8
-export RH_NO_DASHBOARD_SUFFIX=without-dashboard
+if [ "$TARGET" == "rh-aut" ]; then
+  export RH_TAG_DIST_SUFFIX=rhcheautomation
```

Comment:
@ScrewTSW could you please point me in what registry this image is pushed ?

Reply:
@ibuziuk `push.registry.devshift.net/osio-prod/rhchestage/rh-che-automation:<upstream_tag>-rhcheautomation-<che_commit_short_hash>`

Reply:
@ScrewTSW am I correct that this is the only registry we are suppose to push rhel based image ?